test/firebase_fcm_test.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:dartbase/dartbase.dart';
+import 'package:dartbase/fcm/message.dart';
+import 'package:test/test.dart';
+
+import 'test_config.dart';
+
+
+
+Future main() async {
+  Firebase firebase;
+  FCM fcm;
+  final nameRegExp = RegExp(r'projects/.*/messages/[0-9]+');
+
+  setUpAll(() async {
+    firebase = await Firebase.initialize(projectId, await ServiceAccount.fromFile(serviceAccountPath));
+
+    await FirebaseAuth.initialize();
+    fcm = FCM(firebase: firebase, fcmConfig: FCMConfig(firebase.projectId));
```

Comment:
Same here, decide if you want a global or local instance. I prefer global personally because it reduces the size of the code and makes things more clear to read

Reply:
I'm going with the same style as `firebase_storage_test.dart`, hopefully with line FirebaseAuth now gone, it's more consistent.